weblab/static/js/experiment_tasks.js

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
You'll need to add a `require()` line for this in the top of `main.js` to get it made available.

Reply:
yes I have put this in now - thanks